core/src/main/java/org/elasticsearch/indices/memory/IndexingMemoryController.java

```diff
@@ -255,49 +251,43 @@ protected void updateShardBuffers(ShardId shardId, ByteSizeValue shardIndexingBu
         }
     }
 
+    protected boolean isShardInactive(ShardId shardId, long inactiveTimeNS) {
```

Comment:
Would `isShardActive` be simpler to understand just because it tests a positive?